quelpa.el

```diff
@@ -951,7 +952,8 @@ Return a cons cell whose `car' is the root and whose `cdr' is the repository."
 This will perform a fast-forward merge or a reset if FORCE."
   (if quelpa-force-upgrade-p
       (quelpa-build--run-process dir "git" "reset" "--hard" ref)
-    (quelpa-build--run-process dir "git" "merge" "--ff-only" ref))
+    (with-demoted-errors "Error: %s"
```

Comment:
This feels wrong to me, because it unconditionally makes errors non-errors.  Would you explain what's going on here?  Is there another way this could be handled?  

Is this even necessary if `git checkout` were used instead of `git merge`?